d2lbook/sphinx_template.py

```diff
@@ -1,6 +1,13 @@
 
 sphinx_conf = r"""
 import sys
+from sphinx.ext.autosummary import Autosummary
+from sphinx.ext.autosummary import get_documenter
+from docutils.parsers.rst import directives
+from sphinx.util.inspect import safe_getattr
```

Comment:
do these imports need to install additional packages?

Reply:
No additional packages.  sphinx built-in